core/txpool/blobpool/blobpool_test.go

```diff
@@ -118,7 +118,7 @@ func (bc *testBlockChain) CurrentBlock() *types.Header {
 
 	for new(big.Int).Add(lo, big.NewInt(1)).Cmp(hi) != 0 {
 		mid := new(big.Int).Add(lo, hi)
-		mid.Div(mid, big.NewInt(2))
+		mid.Rsh(mid, 1)
```

Comment:
Pls revert these two. these are in tests, so I prefer readability vs performance.

Reply:
reverted